fdbserver/KeyValueStoreMemory.actor.cpp

```diff
@@ -462,39 +462,56 @@ class KeyValueStoreMemory final : public IKeyValueStore, NonCopyable {
 		return total;
 	}
 
-	// Data format for normal operation:
+	static bool isOpEncrypted(OpHeader* header) { return header->op >> ENCRYPTION_ENABLED_BIT == 1; }
+
+	static void setEncryptFlag(OpHeader* header, bool set) {
+		if (set) {
+			header->op |= (1UL << ENCRYPTION_ENABLED_BIT);
+		} else {
+			header->op &= ~(1UL << ENCRYPTION_ENABLED_BIT);
+		}
+	}
+
+	// NOTE: The first byte of opType indicates whether the entry is encrypted or not. This is fine for backwards
+	// compatability since the first byte was never used previously
+	//
+	// Unencrypted data format (1 byte padding at the end):
 	// +-------------+-------------+-------------+--------+--------+
 	// | opType      | len1        | len2        | param2 | param2 |
 	// | sizeof(int) | sizeof(int) | sizeof(int) | len1   | len2   |
 	// +-------------+-------------+-------------+--------+--------+
 	//
-	// However, if the operation is encrypted:
-	// +-------------+-------------+-------------+---------------------------------+-------------+--------+--------+
-	// | OpEncrypted | len1        | len2        | BlobCipherEncryptHeader         | opType      | param1 | param2 |
-	// | sizeof(int) | sizeof(int) | sizeof(int) | sizeof(BlobCipherEncryptHeader) | sizeof(int) | len1   | len2   |
-	// +-------------+-------------+-------------+---------------------------------+-------------+--------+--------+
-	// |                                plaintext                                  |           encrypted           |
-	// +-----------------------------------------------------------------------------------------------------------+
+	// Encrypted data format (1 byte padding at the end):
```

Comment:
Can you help update the illustration to include the `zeroFilled` byte at the end? for both encrypted and unencrypted format.